docs/products/kafka/kafka-connect/howto/s3-source-connector.md

```diff
@@ -0,0 +1,193 @@
+---
```

Comment:
In the packaging pr, we have docs link provided as https://github.com/aiven/aiven-core/pull/55517/files#diff-081e88de4c9f25b03afcdc10257834c5bc6ec2e3a8176749fc283e759441f017R113

similar to s3 sink. Possible to change this file name ?

Reply:
Or I can change my packaging pr too.